px-map-behavior-marker-group.es6.js

```diff
@@ -670,6 +672,27 @@
       this._bindAndOpenPopup(evt.layer);
     },
 
+    _handleSingleMarkerDoubleTap(evt) {
+      const latLng = L.latLng(evt.layer.getLatLng());
+      const {lat, lng} = latLng;
+      const detail = {
+        latLng: latLng,
+        lat: lat,
+        lng: lng
+      };
+      this.fire('px-map-marker-group-double-tapped', detail);
+    },
+    /**
+     * Fired when the marker is double clicked or tapped by the user.
```

Comment:
change to `Fired when an individual marker is double clicked by the user. Not fired when a marker cluster is double clicked. Note that this will only work on non-touch (e.g. desktop) devices. Touch (e.g. mobile) devices cannot respond to double click/tap events.`